data/cookbook/invert-matrix/01-owl.ml

```diff
@@ -0,0 +1,35 @@
+---
+packages: [owl]
+---
+
+(* Creates an Owl `float` matrix using random numbers drawn from a uniform distribution with `Owl.Mat.uniform`. It is very likely that it will be invertible but we ensure it by checking that the determinat is not zero with `Owl.Linalg.D.dat`. The dimensions of the matrix are defined by `m`. *)
```

Comment:
```suggestion
(* Creates an Owl `float` matrix using random numbers drawn from a uniform distribution with `Owl.Mat.uniform`. It is very likely that it will be invertible, but we ensure it by checking that the determinat is not zero with `Owl.Linalg.D.dat`. The dimensions of the matrix are defined by `m`. *)
```
Should the word be "determinant" or is "determinat" correct?